src/docfx/config/ContributorConfig.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Docs.Build
+{
+    internal class ContributorConfig
+    {
+        /// <summary>
```

Comment:
There should be a flag to enable/disable contribution information

Reply:
will add it in later PR

---
In reply to: [197009200](https://github.com/dotnet/docfx/pull/2912#discussion_r197009200) [](ancestors = 197009200)